### PR TITLE
JitArm64: Implement subfex, subfcx, addex, subfic, divwux, srwx

### PR DESCRIPTION
**Pull request by degasus**

### CODE REVIEW

File: Source/Core/Core/PowerPC/JitArm64/JitArm64_Integer.cpp [hunk 1]
```diff
@@ -724,6 +724,188 @@ void JitArm64::subfx(UGeckoInstruction inst)
...
+		}
+	}
+	else
+	{
+		ARM64Reg WA = gpr.GetReg();
+		gpr.BindToRegister(d, d == a || d == b);
+
+		// upload the carry state
+		LDRB(INDEX_UNSIGNED, WA, X29, PPCSTATE_OFF(xer_ca));
+		SUBS(WZR, WA, 1);
```
Comment (Sonicadvance1) [inline Source/Core/Core/PowerPC/JitArm64/JitArm64_Integer.cpp]: You should use the CMP alias here

File: Source/Core/Core/PowerPC/JitArm64/JitArm64_Integer.cpp [hunk 2]
```diff
@@ -724,6 +724,188 @@ void JitArm64::subfx(UGeckoInstruction inst)
...
+		}
+	}
+	else
+	{
+		gpr.BindToRegister(d, d == a || d == b);
+
+		// upload the carry state
+		ARM64Reg WA = gpr.GetReg();
+		LDRB(INDEX_UNSIGNED, WA, X29, PPCSTATE_OFF(xer_ca));
+		SUBS(WZR, WA, 1);
```
Comment (Sonicadvance1) [inline Source/Core/Core/PowerPC/JitArm64/JitArm64_Integer.cpp]: You should use the CMP alias here

File: Source/Core/Core/PowerPC/JitArm64/JitArm64_Integer.cpp [hunk 3]
```diff
@@ -801,6 +1011,52 @@ void JitArm64::slwx(UGeckoInstruction inst)
...
+		if (amount & 0x20)
+		{
+			gpr.SetImmediate(a, 0);
+			if (inst.Rc)
+				ComputeRC(0, 0);
+		}
+		else
+		{
+			gpr.BindToRegister(a, a == s);
+			LSR(gpr.R(a), gpr.R(a), amount & 0x1F);
```
Comment (degasus) [inline Source/Core/Core/PowerPC/JitArm64/JitArm64_Integer.cpp]: a>>b, not a>>a

**Comment by Sonicadvance1** [comment]
LGTM

**Pull request merged**